bin/cli.js

```diff
@@ -37,7 +37,8 @@ function getLatestTag() {
 }
 
 function getParentRevision() {
-    return execSync(`hg log --rev "parents(.)" --template "{rev}" `);
+    const currentBranch = getCurrentBranch();
+    return execSync(`hg log --rev "parents(.)" --template "{rev}" -b ${currentBranch} `);
```

Comment:
I think you're missing the `.toString().trim();`, but it wasn't there before so maybe it's not needed

You also have a trailing space at the end of the string